### PR TITLE
Fixes errcheck issues in server/channels/api4/team_test.go

### PR DESCRIPTION
**Pull request by azadDsync**
#### Ticket Link
Fixes https://github.com/mattermost/mattermost/issues/28562

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Comment by azadDsync** [comment]
@hanzei, I need help fixing the failing checks in Enterprise CI/tests.

File: server/channels/api4/team_test.go [hunk 1]
```diff
@@ -2298,8 +2322,8 @@ func TestAddTeamMember(t *testing.T) {
 	_, resp, err = client.AddTeamMemberFromInvite(context.Background(), token.Token, "")
 	require.Error(t, err)
 	CheckNotFoundStatus(t, resp)
-	th.App.DeleteToken(token)
-
+	appErr = th.App.DeleteToken(token)
+	require.Nil(t, appErr)
 	// by invite_id
```
Comment (hanzei) [inline server/channels/api4/team_test.go:2327]: Nit: Let's add back the newline
```suggestion
	require.Nil(t, appErr)
	// by invite_id

```

**Review by hanzei — APPROVED** [review]
Great work :tada:

**Review by nickmisasi — APPROVED** [review]
LGTM. Thanks @azadDsync 🚀

**Pull request merged**